canonical-web-and-design/maas-ui
# Add basic DNS listing page

## Done

- Added basic routing and components for DNS listing page

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- run `yarn ui`
- go to http://0.0.0.0:8400/MAAS/r/domains
- DNS page should be served with basic header component

## Fixes

Fixes: https://github.com/canonical-web-and-design/app-squad/issues/34
